### PR TITLE
Change conditional build logic

### LINKED ISSUE #103: Deploy tests are not being triggered
It seems that the deploy stage is not being triggered in Travis. 

Here is an example of a build from a PR that should have triggered the deployment.

https://travis-ci.com/lagom/lagom-samples/builds/130444259

Not PR is NOT sent from a fork and is targeting on the main branches (ie: 1.5.x)

### PR DESCRIPTION
**Pull request by TimMoore**
The encrypted vars don't appear to be available when evaluating these conditions.

Resolves #103

### CODE REVIEW

**Review by dwijnand — APPROVED** [review]
lgtm

**Comment by TimMoore** [comment]
The build failures are unrelated problems that were exposed by this change.

**Comment by ignasi35** [comment]
The build is broken because `jabba`.  Needs #107

**Comment by TimMoore** [comment]
All of the test stages passed, @ignasi35, it's only the deployment stages that are failing. Before this pull request, they weren't run at all for some time.

**Comment by ignasi35** [comment]
> All of the test stages passed, @ignasi35, it's only the deployment stages that are failing. Before this pull request, they weren't run at all for some time.

Let me rephrase:

 - the deploy jobs had a failure when trying to install `jabba` (via `$ . bin/jabbaLib && installJabba`) which is now fixed in #107
 - fixing #107 still doesn't resolve the build as @TimMoore  pointed out

I only wanted to highlight there was an extra failure in the logs. :-)

**Comment by TimMoore** [comment]
This has a green build now.
